src/app/core/services/misc/github-client.service.ts

```diff
@@ -8,11 +9,15 @@ import { AppServerService } from '../app-server/app-server.service'
 export class GithubClient {
   constructor(
     private appServerService: AppServerService,
-    private util: Utility
+    private util: Utility,
+    private http: HttpClient
   ) {}
 
   getRaw(url): Promise<any> {
-    return this.appServerService.fetchFromGithub(url)
+    return this.appServerService.fetchFromGithub(url).catch(e => {
+      if (e.status == 404) return this.http.get(url).toPromise()
```

Comment:
what happens if the repo is private? I think it would be better to control this via a property on firebase remote config, similar to how we handle the REST and XMPP notifications. So we can set for a deployment base URL or a project to use public github URLs instead of the appserver and vice-versa. I think the default should remain the public git URLs for now.

Reply:
Okay sure will make the changes.